app/models/config-model.js

```diff
@@ -17,7 +17,11 @@ const execSync = require( 'child_process' ).execSync;
 
 // Merge default and local config files if a local config.json file exists
 try {
-    const localConfig = require( '../../config/config' );
+    const localConfigJSON = String(
+        fs.readFileSync( path.resolve( process.cwd(), './config/config.json' ) )
```

Comment:
Why is this changing? I vaguely remember some back and forth in how the base path was determined in various contexts, does this account for that.

Reply:
Just so it’s mockable in test. Sinon can’t mock a bare function call, and `require` isn’t really a global (it’s an argument to an implied function that wraps each CommonJS module). Other test frameworks like Jest allow mocking whole modules with various forms of chicanery, but it’s… a whole mess in itself.